### PR TITLE
Issue 10/add packaging

### PR DESCRIPTION
**Pull request by jekwatt**
Begin with bare minimum packaging
Use underscore to name package `pm_utils`
Add `pandas`, `xlrd`,  `openpyxl` to REQUIREMENTS in `setup.py`
Generate expected output files in data directory

### CODE REVIEW

**Review by itsmarcy — APPROVED** [review]
Looks good to me!
Awesome job!

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+    ],
+    packages=find_packages(exclude=["contrib", "docs", "tests"]),
+    install_requires=REQUIREMENTS,
+    tests_requires=TEST_REQUIREMENTS,
```
Comment (walkerh) [inline setup.py]: The `tests_requires` doesn't really get used, so you don't get what you want. Right now, that doesn't matter, but you would need to manually install `pytest` if you plan on using such tests.

If you aren't planning on using `pytest`, just drop the `tests_requires=TEST_REQUIREMENTS,` line and the paragraph above that defines `TEST_REQUIREMENTS`.

Later, if you want to add support for `pytest` add this code:

```python
TEST_REQUIREMENTS = [
    "pytest",
]

EXTRAS_REQUIRE = {  # Optional
    'test': TEST_REQUIREMENTS,
}

...
setup(
    ...
    extras_require=EXTRAS_REQUIRE,
    ...
)
```

See this [example](/BCM-HGSC/bhaou_catapult/blob/32bc6ee369da3f809b23dd257f4606b33a78a7c9/setup.py#L48)

**Review by walkerh — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (jekwatt) [reply setup.py]: That is my next item to do.  I plan to create another branch for issue #11 and add tests.

Comment (walkerh) [reply setup.py]: Then I suggest deleting line 53 (`tests_requires`), since it doesn't do anything useful.

Comment (jekwatt) [reply setup.py]: True.  Removing the unnecessary requirements.

**Review by walkerh — APPROVED** [review]
Works for me.

**Comment by jekwatt** [comment]
This PR has been merged

**Pull request merged**